src/scripts/worldmap/MapHelper.ts

```diff
@@ -248,6 +245,26 @@ class MapHelper {
         }
     }
 
+    public static accessToShip() {
+        return player.highestRegion() > 0 && TownList[GameConstants.DockTowns[player.region]].isUnlocked();
+    }
+
+    // For moving between already-unlocked regions
+    public static moveToRegion(region: GameConstants.Region) {
+        if (MapHelper.accessToShip()) {
+            if (region <= player.highestRegion() && region <= GameConstants.MAX_AVAILABLE_REGION && region !== GameConstants.Region.none) {
+                player.region = region;
+                player._subregion(0);
```

Comment:
`moveToTown` already does [that](https://github.com/pokeclicker/pokeclicker/blob/8ff2820f3672f316748dc3fbec4f5590d92ee3c7/src/scripts/worldmap/MapHelper.ts#L212) so that line can be removed safely. Same comment applies to line 287.